gradle.properties

```diff
@@ -40,7 +40,8 @@ doUploadUBenchResults=false
 #
 # Internal Nexus location
 #
-localNexusLocation=http://nexus.0xdata.loc:8081/nexus/repository
```

Comment:
this keeps changing back and forth, recently Honza changed it to the full URL, is there a reason not to keep it as-is?

Reply:
ok, will revert: thought this was old due to the domain name